provider/cmd/pulumi-resource-command/schema.json

```diff
@@ -217,6 +217,11 @@
           },
           "description": "The program and arguments to run the command.\nOn Linux and macOS, defaults to: `[\"/bin/sh\", \"-c\"]`. On Windows, defaults to: `[\"cmd\", \"/C\"]`"
         },
+        "logOutput": {
+          "type": "boolean",
+          "description": "If the command's stdout and stderr should be logged.",
```

Comment:
I wonder if users might wanna select what streams end up being logged, e.g. stderr only.

By using a boolean we kind of limit the API for the future